.changeset/tiny-seals-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/design-tokens": minor
+---
+
```

Comment:
Should we have it is only docs?

Reply:
@nyan07 Can you explain more? I wasn't sure about the changeset here. Mostly, the docs components are importing from the components folder. So should the localyze-pluto/components have a minor update here?

Reply:
We don't need to generate a new version because it doesn't change anything on components that are exported to usage. I believe the idea here is to only focus on docs and not change the bundle. So, it is just a change on the repo, not a new version.